### ISSUE TITLE
Add a disable function, like react-komposer

### ISSUE THREAD

**Issue by zeroasterisk** [studiointeract/tracker-component#12]
Justification: easy container testing outside of Meteor
https://github.com/kadirahq/react-storybook/blob/master/docs/configure_storybook.md#configure-modules-for-testing

**Comment by timbrandin** [comment]
Alright this should be simple to add, is the use-case mainly for React Storybook?

**Comment by zeroasterisk** [comment]
Yeah - that's the only thing I can think of... but since Storybook publishes to github pages, I've been using it more and more for all non-meteor react stuff.

Low priority... but as you said, should be super easy.
